.changeset/early-experts-wave.md

```diff
@@ -0,0 +1,5 @@
+---
+'@microsoft/atlas-site': patch
+---
+
+Cleaning up the hero markup. Removing an unneeded class and adding an atomic for the hero-details use case.
```

Comment:
What about "Updated markup for hero-details"?

Reply:
Done